src/devices/CharacterLcd/GpioControllerAdapter.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio;
+
+namespace Iot.Device.CharacterLcd
+{
+    public class GpioControllerAdapter : IGpioController
```

Comment:
Can you log an issue to remove these two adapters once IGpioController is pulled up to the main library and implemented by both GpioController and Mcp?

Reply:
Yes, will do